content/en/docs/reference/config-api/apiserver-audit.v1.md

```diff
@@ -253,7 +253,7 @@ be specified per rule in which case the union of both are omitted.</p>
 <td>
    <p>OmitManagedFields indicates whether to omit the managed fields of the request
 and response bodies from being written to the API audit log.
-This is used as a global default - a value of 'true' will omit the managed fileds,
+This is used as a global default - a value of 'true' will omit the managed fields,
```

Comment:
This [kube-apiserver Audit Configuration](https://kubernetes.io/docs/reference/config-api/apiserver-audit.v1/) reference page is [auto-generated](https://kubernetes.io/docs/reference/config-api/apiserver-audit.v1///#auto-generated-edit-notice).

Since the reference page is automatically generated instead of manually fixing the contents, consider [regenerating](https://kubernetes.io/docs/contribute/generate-ref-docs/) the contents of this page